spec/lib/user_helpers_spec.rb

```diff
@@ -45,7 +45,7 @@
       it { expect(user.to_resource_h)
                .to json_eq(
                        uid: json[:uid],
-                       profile_picture: json[:image_url],
+                       image_url: json[:image_url],
```

Comment:
Could we just keep both? Having avatars would be nice to https://github.com/mumuki/mumuki-domain/pull/90

Reply:
Take into account that image_url is defined as `profile_picture` in resource_h which does use avatars.
For now no avatar is set, but that's because we wanted to avoid having to create new avatars in the db right before this one is _integrated_.